i18n/locales/es.ts

```diff
@@ -0,0 +1,376 @@
+      title: 'Génesis',
+      heading: 'Tu primer reto',
+      paragraph_one:
+        'Bitcoin es dinero resistente a la censura. Cualquiera puede enviar dinero emitiendo una transacción a la red. Tras la emisión, los mineros empaquetan las transacciones en bloques. Los mineros compiten contra otros mineros por el privilegio de construir en la cadena. Esto es lo que mantiene a bitcoin descentralizado',
+      paragraph_two:
+        'Satoshi Nakamoto, el inventor seudónimo de bitcoin, también minó el primer bloque de bitcoin. Dejó al mundo un mensaje secreto en la primera transacción bitcoin que se hizo. Tu primer reto es encontrarlo y descifrarlo',
+    },
+
+    genesis_two: {
+      title: 'Génesis',
+      heading: 'Encuentra el mensaje oculto',
+      paragraph_one:
+        'Vamos a encontrar el primer bloque de la blockchain de bitcoin. Haz clic en el botón de abajo para abrir un <Tooltip id="genesis_two_paragraph_one" content="genesis_two.tooltip_block_explorer" className="underline">explorador de bloques</Tooltip> exactamente en el bloque 0, que se llama bloque génesis.',
+      paragraph_two:
+        'Desplázate hacia abajo y expande los detalles de la única transacción que se almacena en este bloque. Busca la entrada llamada "Coinbase". Ahora busque la etiqueta "SCRIPTSIG (<Tooltip id="genesis_two_paragraph_two" content="genesis_two.tooltip_hex" className="underline">HEX</Tooltip>)". El valor que aparece junto a él es un mensaje codificado',
+      paragraph_three: 'Copia ese valor y pégalo debajo.',
+      tooltip_block_explorer:
+        'A <a href="https://bitcoinops.org/en/topics/block-explorers/" target="_blank" rel="noreferrer">block explorer</a> is a useful tool to quickly look up information about bitcoin transactions.',
+      tooltip_hex:
+        'Abreviatura de hexadecimal, un sistema numérico que representa números utilizando una base de 16.',
+      view_block_0: 'Ver bloque 0',
+      placeholder: 'Pega el valor que has encontrado aquí',
+    },
+
+    genesis_three: {
+      title: 'Génesis',
+      heading: 'Descifremos el mensaje',
+      paragraph_one:
+        'El mensaje que encontraste estaba codificado en un formato llamado HEX. Ahora ejecutaremos un comando para convertirlo en ASCII, que podremos leer',
+      paragraph_two:
+        'Copia y pega el comando de abajo en el Terminal de la derecha y pulsa "Enter"',
+      terminal_challenge_lines: 'Introduzca sus comandos aquí y pulse Intro...\n\n La variable $scriptSigHex ya está definida para usted.\n\n var $scriptSigHex = "04fff...e6b73"',
+      waiting_for_input: 'Esperando a que escribas y ejecutes el script...',
+      éxito: 'Buen trabajo! El mensaje descifrado hace referencia a la portada de <Link href="https://en.bitcoin.it/wiki/Genesis_block" className="underline">The Times</Link> del 3 de enero de 2009, el mismo día que Satoshi minó el bloque génesis. ¿No es genial? Este mensaje también nos da una idea de su motivación para crear bitcoin,',
```

Comment:
My guess is that the thing to the left of the colon is some kind of key and should not be translated. Missing opening exclamation mark:

```suggestion
      success: '¡Buen trabajo! El mensaje descifrado hace referencia a la portada de <Link href="https://en.bitcoin.it/wiki/Genesis_block" className="underline">The Times</Link> del 3 de enero de 2009, el mismo día que Satoshi minó el bloque génesis. ¿No es genial? Este mensaje también nos da una idea de su motivación para crear bitcoin,',
```